ddtrace/contrib/internal/aiohttp/patch.py

```diff
@@ -31,7 +31,12 @@
 # Server config
 config._add(
     "aiohttp",
-    dict(distributed_tracing=True),
+    dict(
+        distributed_tracing=True,
+        disable_stream_timing_for_mem_leak=asbool(
+            os.getenv("DD_AIOHTTP_CLIENT_DISABLE_STREAM_TIMING_FOR_MEM_LEAK", default=False)
```

Comment:
Any way to change this to use `_get_config`?

Reply:
I don’t think we use get_config for any of our integration specific configurations? @tabgok are we trying to move towards sending all configs, including integration specific ones via telemetry?

Reply:
Regardless of the telemetry goals, it's important to limit the use of `os.getenv` because it makes configs harder to track and centralize.